lib/netuitived/metric_aggregator.rb

```diff
@@ -53,7 +54,12 @@ def sendMetrics
           aggregatedSamplesArray.each do |sample|
             sample.timestamp = Time.new
           end
-          element = NetuitiveD::IngestElement.new(NetuitiveD::ConfigManager.elementName, NetuitiveD::ConfigManager.elementName, 'Ruby', nil, @metrics, @samples + aggregatedSamplesArray, nil, nil)
+	        unless NetuitiveD::ConfigManager.elementTags.nil?
+            for tag in NetuitiveD::ConfigManager.elementTags.split(',').map(&:strip) do
```

Comment:
What do you think about adding a try/catch around all this? A bad config will prevent startup. Maybe that's the best scenario, I'm not really sure. E.g. `name1:value1:name2:value2`.

I like the removal of whitespace, too.

Reply:
name1:value1:name2:value2 doesn't prevent startup. name2:value2 is dropped so an element with only name1:value1 is created.

I see where your head is at though. Can you think of another bad configs that might warrant a begin/rescue?

Reply:
Ah, I see. `name` would break during the second array access.

Reply:
`name` creates a tag name `name` with a null value, which is fine I think.

Reply:
I'm going ahead and proceeding with things as is. We can consider adding try/catch logic in a future iteration.